.github/workflows/python-sdk-publish.yaml

```diff
@@ -0,0 +1,43 @@
+name: "Python: Publish @python/serverless-sdk"
```

Comment:
@alexdelorenzo this PR should not introduce those changes.

Can you ensure it contains only changes related to SC-260?

Reply:
@medikoo Good catch, I've cleaned up this PR to reflect only the changes for SC-260